service/utility.py

```diff
@@ -6,7 +6,8 @@
 
 CHUNK_SIZE = 1024 * 1024
 block_blob_service = BlockBlobService(account_name=config.CONFIG["storage"],
-                                      account_key=config.CONFIG["storage_key"])
+                                      account_key=config.CONFIG["storage_key"],
+                                      connection_string="BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;")
```

Comment:
Do we know what is the default value?

I suggest using a config file + environment variable approach just like the others.